b2b_ecommerce/urls.py

```diff
@@ -12,6 +17,14 @@
         B2BOrderFulfillmentView.as_view(),
         name="b2b-order-fulfillment",
     ),
+    url(
+        "^api/b2b/order_enrollment_codes/$",
```

Comment:
Might be more sensible to change this to take the hash as a part of the route and shorten the resource name: `r"^api/b2b/codes/(?P<hash>\w+)$"`